Sloth.Web/Handlers/VerifyTeamPermissionHandler.cs

```diff
@@ -27,6 +27,9 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
         {
             if (context.User.IsInRole(Roles.SystemAdmin))
             {
+                // sys admin gets all available roles
+                _httpContext.HttpContext.Items.Add("TeamRoles", new [] { TeamRole.Admin, TeamRole.Approver });
```

Comment:
And I guess if we have system and want to just see certain stuff we emulate...

Reply:
yea, this means system admins can see/do everything.  which i think is what we want.  otherwise we couldn't go look at transactions or team configs